app/static/lib/conp-react/src/CbrainModalPipeline/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from "react";
+import * as ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+const CbrainModalPipeline = (props) => {
+  const { platforms, title, cbrainIds } = props;
+  const finish = (event) => {
+    $("#cbrainModal").modal("hide");
+  };
+  const baseUrl = platforms[0].uri;
+  const getDatasetId = (datasetUrl) =>
+    `switch_group_id=${datasetUrl.split("?id=")[1]}`;
+  useEffect(() => {
+    $("#cbrainModal").modal("handleUpdate");
+    return;
+  });
+  return ReactDOM.createPortal(
+    <div>
+      <p>
+        You're about to load the pipeline <strong>{title}</strong> on
```

Comment:
```suggestion
        You're about to launch <strong>{title}</strong> on
```

Reply:
same comment as for the dataset but used the word launch instead of load (load for dataset and launch for pipeline seemed logical to me but does not mean it is necessarily)